spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -370,12 +370,12 @@ The following table includes the `dataflow.spring.io` links to the task applicat
 |Artifact Type |Stable Release |SNAPSHOT Release
 
 |Maven
-|https://dataflow.spring.io/task-maven-latest
-|https://dataflow.spring.io/task-maven-latest-snapshot
+|https://dataflow.spring.io/task-maven-3-0-x
```

Comment:
Further digging reveals this is ok because the default SCDF configuration includes the anonymously accessible `repo.spring.io/milestone` where the task samples are available.